src/github.com/getlantern/lantern-mobile/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# Dockerfile to build an image able to compile flashlight for iOS and Android
+#
+# > make docker
+
+FROM fedora:22
+MAINTAINER "Ulysses Aalto" <uaalto@getlantern.org>
+
+ENV GO_VERSION go1.4.2
+ENV GOROOT_BOOTSTRAP /go1.4
+ENV GOROOT /go
+ENV GOPATH /
+
+ENV PATH $PATH:$GOROOT/bin
+ENV WORKDIR /lantern
+
+# Updating system.
+RUN dnf -y update && dnf clean all
+
+# Requisites for building Go.
+RUN dnf install -y git tar patch gzip curl hostname pcre-devel && dnf clean all
```

Comment:
It is not caused by a missing `which` utility, but rather by `which` not finding a file.
I tried to improve this, but I think is not worth fiddling with it: the issue seems to be with their testing scripts. They test the existence of executable which is part of the test suite. Anyway these error recover, and are internal to the test suite.
We can also avoid running the test suite at all.
There were other errors before caused by missing programs, but I already fixed them before the pull request.